src/modules/mobility/components/filter/MobilityFilters.tsx

```diff
@@ -36,38 +37,53 @@ export const MobilityFilters = ({filter, onFilterChanged}: Props) => {
       onFilterChanged(newFilter);
     };
 
-  return (
-    <View style={styles.container}>
-      {isVehiclesInMapEnabled && (
-        <FormFactorFilter
-          formFactor={FormFactor.Scooter}
-          icon={Scooter}
-          initialFilter={filter[FormFactor.Scooter]}
-          onFilterChange={onFormFactorFilterChanged(FormFactor.Scooter)}
-        />
-      )}
-      {isCityBikesInMapEnabled && (
-        <FormFactorFilter
-          formFactor={FormFactor.Bicycle}
-          icon={Bicycle}
-          initialFilter={filter[FormFactor.Bicycle]}
-          onFilterChange={onFormFactorFilterChanged(FormFactor.Bicycle)}
-        />
-      )}
-      {isCarSharingInMapEnabled && (
-        <FormFactorFilter
-          formFactor={FormFactor.Car}
-          icon={Car}
-          initialFilter={filter[FormFactor.Car]}
-          onFilterChange={onFormFactorFilterChanged(FormFactor.Car)}
-        />
-      )}
-    </View>
+  const listedFormFactors: FormFactor[] = [
+    {
+      shouldShow: isVehiclesInMapEnabled,
+      formFactor: FormFactor.Scooter,
+    },
+    {
+      shouldShow: isCityBikesInMapEnabled,
+      formFactor: FormFactor.Bicycle,
+    },
+    {
+      shouldShow: isCarSharingInMapEnabled,
+      formFactor: FormFactor.Car,
+    },
+  ]
+    .filter((f) => f.shouldShow)
+    .map((f) => f.formFactor);
+
+  const FormFactors = listedFormFactors.map((formFactor, i) => (
+    <FormFactorFilter
+      key={formFactor}
+      isFirstSectionItem={i === 0}
+      isLastSectionItem={i === listedFormFactors.length - 1}
+      formFactor={formFactor}
+      initialFilter={filter[formFactor]}
+      onFilterChange={onFormFactorFilterChanged(formFactor)}
+    />
+  ));
+
+  return isMapV2Enabled ? (
+    <Section>
+      <ContentHeading
+        text={t(MobilityTexts.filter.sectionTitle.sharedMobility)}
+        style={styles.contentHeading}
+      />
+      {FormFactors}
+    </Section>
```

Comment:
Should move contentHeading outside Section
```suggestion
    <>
    <ContentHeading
        text={t(MobilityTexts.filter.sectionTitle.sharedMobility)}
        style={styles.contentHeading}
      />
    <Section>
      {FormFactors}
    </Section>
    </>
```